includes/admin/settings/class-wcmpbe-settings-data.php

```diff
@@ -413,6 +413,15 @@ private function get_section_general_general(): array
                 "type"      => "toggle",
                 "help_text" => __("Show Track & Trace trace code and link in My Account.", "woocommerce-myparcelbe"),
             ],
+            [
+                "name"      => WCMPBE_Settings::SETTING_SHOW_DELIVERY_DAY,
+                "label"     => __("Show delivery date", "woocommerce-myparcelbe"),
+                "type"      => "toggle",
+                "help_text" => __(
+                    "Show delivery day options allow your customers to see the delivery day in order confirmation and My Account.",
```

Comment:
Waarom is dit niet meteen een translation key?

Reply:
En gebruik hier dan ook meteen single quotes als je toch bezig bent :P

Reply:
Ja zat ik ook aan te denken... ok.